ostd/src/coverage.rs

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
Add the following module-level Rust doc to describe the exact meaning of "coverage".

```rust
//! Support for the code coverage feature of OSDK.
//!
//! For more information about the code coverage feature (`cargo osdk run --coverage`),
//! check out the OSDK reference manual.
```